package.json

```diff
@@ -10,10 +10,10 @@
   },
   "dependencies": {
     "@babel/runtime": "7.3.1",
-    "@patternfly/patternfly-next": "1.0.171",
-    "@patternfly/react-core": "1.51.4",
-    "@patternfly/react-icons": "2.10.3",
-    "@red-hat-insights/insights-frontend-components": "3.33.48",
+    "@patternfly/patternfly-next": "1.0.175",
```

Comment:
Just FYI, we just found out that this will be moving to `@patternfly/patternfly`

Reply:
:+1: